packages/commonwealth/vite.config.ts

```diff
@@ -0,0 +1,124 @@
+import react from '@vitejs/plugin-react-swc';
+import path from 'path';
+import { Alias, defineConfig, loadEnv } from 'vite';
+import { createHtmlPlugin } from 'vite-plugin-html';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+const projectRootDir = path.resolve(__dirname);
+
+function createClientResolver(folder: string): Alias {
+  const find = new RegExp(`^${folder}/(.*)$`);
+  return {
+    find,
+    replacement: path.resolve(
+      projectRootDir,
+      'client',
+      'scripts',
+      folder,
+      '$1',
+    ),
+  };
+}
+
+// https://vitejs.dev/config/
+export default defineConfig(({ command, mode }) => {
+  const env = loadEnv(mode, process.cwd(), '');
+  return {
+    plugins: [
+      react(),
+      createHtmlPlugin({
+        entry: `./${command === 'serve' ? 'client/' : ''}scripts/index.tsx`,
+        template: './client/index.html',
+      }),
+      tsconfigPaths(),
+    ],
+    publicDir: 'static',
```

Comment:
getting this when accessing some image content
 
<img width="1235" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/0df8c5c4-2ece-42c3-9115-c62d8dbcdcc2">